doc/api/console.md

```diff
@@ -332,6 +332,47 @@ console.log('count:', count);
 
 See [`util.format()`][] for more information.
 
+### console.table(tabularData[, properties])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `tabularData` {any}
+* `properties` {Array<String>} Alternate properties for constructing the table.
+
+Try to construct a table with the columns of the properties of `tabularData`
+(or use `properties`) and rows of `tabularData` and logit. Falls back to just
+logging the argument if it can’t be parsed as tabular.
+
+<!-- eslint-skip -->
+```js
+// These can't be parsed as tabular data
+> console.table(Symbol())
+Symbol()
+> console.table(undefined)
+undefined
```

Comment:
Can you please comment the output? That is always done in in other places to mark output :-)

You can also write e.g., "Prints:" in-front of it if you like.

That way the eslint comment can also be removed.